docs/developer-guide/integrations/users/openId.md

```diff
@@ -98,11 +97,11 @@ googleOAuth2Config.discoveryUrl=https://accounts.google.com/.well-known/openid-c
 
 [Keycloak](https://www.keycloak.org/) is an open source identity and access management application widely used. MapStore has the ability to integrate with keycloak:
 
-- Using the standard OpenID protocol
-- Supporting SSO (not yet implemented)
-- Integrating with users and roles, as well as for ldap. (not yet implemented)
+- Using the standard OpenID Connect (OIDC) protocol to login/logout in MapStore
+- Supporting Single Sign On (SSO) with other applications.
+- Mapping keycloak roles to MapStore groups, as well as for ldap.
 
-In this section you can see how to configure keycloak as a standard OpenID provider
+In this section you can see how to configure keycloak as a standard OpenID provider. For the other advanced functionalities, you can see the [dedicated section of the documentation](../keycloak)
```

Comment:
In this section you can see how to configure keycloak as a standard OpenID provider. For other advanced functionalities, you can see the [dedicated section of the documentation](../keycloak)

Reply:
```suggestion
In this section you can see how to configure keycloak as a standard OpenID provider. For other advanced functionalities, you can see the [dedicated section of the documentation](../keycloak)
```